### PR TITLE
samples: bluetooth: Add Apple iBeacon demo application

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
Add a simple demo implementation of an Apple iBeacon BLE broadcaster.

Signed-off-by: Henrik Brix Andersen <henrik@brixandersen.dk>

### CODE REVIEW

**Comment by henrikbrixandersen** [comment]
recheck

**Review by jhedberg — APPROVED** [review]

File: samples/bluetooth/ibeacon/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. _bluetooth-ibeacon-sample:
+
+Bluetooth: iBeacon
+##################
+
+Overview
+********
+
+A simple application demonstrating the BLE Broadcaster role
```
Comment (dbkinder) [inline samples/bluetooth/ibeacon/README.rst]: Let's make this a sentence:

    This simple application demonstrates the BLE ...

**Review by dbkinder — CHANGES_REQUESTED** [review]
One little tweak, otherwise looks OK.

**Comment by henrikbrixandersen** [comment]
@dbkinder Good suggestion, thanks. I have changed it into a sentence.

**Review by Vudentz — APPROVED** [review]
LGTM

**Comment by jhedberg** [comment]
I'll merge this since it seems the change request from @dbkinder has been taken care of.

**Pull request merged**